plugins/obs-qsv11/QSV_Encoder.h

```diff
@@ -57,8 +57,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #pragma once
 
 #include <Windows.h>
-#include "mfxstructures.h"
-#include "mfxadapter.h"
+#include "vpl/mfxstructures.h"
```

Comment:
Now that we are not using an in source copy of the VPL library please make sure these are system includes instead of relative includes.

Reply:
Should be fixed now!